### PR TITLE
add mongo note to changelog

### PR DESCRIPTION
**Pull request by punkrokk**
added a note to the changelog regarding mongo 4.0 for EL7 and U16

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -29,6 +29,8 @@ Fixed
 
 * Fixed a bug in the example nginx HA template declared headers twice (bug fix) #4966
   Contributed by @punkrokk
+  
+* Switched mongo 4.0 as the default version in EL8 and Ubuntu 16.04 (xenail) @punkrokk
```
Comment (nmaludy) [inline CHANGELOG.rst:33]: Isn't this going to be the default for all OSes?

Comment (armab) [reply CHANGELOG.rst:33]: Right.

Also in the Changelog `Changed` category is more suitable for this change, comparing to `Fixed` category.

File: CHANGELOG.rst [hunk 2]
```diff
@@ -9,6 +9,11 @@ Added
 * Add make command to autogen JSON schema from the models of action, rule, etc. Add check
   to ensure update to the models require schema to be regenerated. (new feature)
 
+Changed
+~~~~~~~
+
+* Switched mongo 4.0 as the default version starting with all supported OS's in ST2 3.3.0 @punkrokk
```
Comment (armab) [inline CHANGELOG.rst:15]: Would be nice to follow the same Changelog format and keep it consistent. Looking at the older records:
```suggestion
* Switch to MongoDB ``4.0`` as the default version starting with all supported OS's in st2 ``v3.3.0`` (improvement) #4972
Contributed by @punkrokk 
```

**Comment by punkrokk** [comment]
@armab Updated.

File: CHANGELOG.rst [hunk 3]
```diff
@@ -9,6 +9,12 @@ Added
 * Add make command to autogen JSON schema from the models of action, rule, etc. Add check
   to ensure update to the models require schema to be regenerated. (new feature)
 
+Changed
+~~~~~~~
+
```
Comment (armab) [inline CHANGELOG.rst:14]:
```suggestion
```

**Comment by punkrokk** [comment]
@armab I'll merge after all of ci/cd runs for removing a new line

**Pull request merged**